torch_xla/core/xrt_run_server.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
```

Comment:
"_xrt_run_server" is blocking(`Join()` is a blocking call) and is the child of the `xla_dist` process. I want to keep the server as a stand alone process and will be running even after `xla_dist` exit.

Reply:
I think if we do 
```
print("python -c \"import subprocess; subprocess.Popen([\'python\', \'-m\', \'torch_xla.core._xrt_run_server\', \'{}\'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, start_new_session=True)\"".format(self.self.tpuvm_server_port))
```

we can get rid of `torch_xla.core.xrt_run_server`, but I felt like it is better to keep this in case user want to start a server on their own for whatever reason. The above workaround is pretty ugly too.